static/css/zamboni/themes.less

```diff
@@ -47,12 +47,6 @@ aside.profile {
 
         + p {
             margin-bottom: 15px;
-
```

Comment:
I think this entire file can go. Theme submission is in devhub which has it's own styles.